helix-query.yaml

```diff
@@ -41,3 +41,28 @@ indices:
       cq-tags:
         select: head > meta[name="cq-tags"]
         value: attribute(el, "content")
+  search:
+    include:
+      - /**
+    exclude:
```

Comment:
@mayur-satav, could you include '/metadata' in this exclude list?

Reply:
Added